workflows/GeneSeqToFamily/GeneSeqToFamily.ga

```diff
@@ -285,48 +408,44 @@
                 }
             ], 
             "position": {
-                "left": 1134.421875, 
-                "top": 380.921875
+                "left": 1240.40625, 
+                "top": 587.40625
             }, 
-            "post_job_actions": {}, 
-            "tool_id": "toolshed.g2.bx.psu.edu/repos/earlhaminst/blast_parser/blast_parser/0.1.1", 
+            "post_job_actions": {
+                "HideDatasetActionoutput": {
+                    "action_arguments": {}, 
+                    "action_type": "HideDatasetAction", 
+                    "output_name": "output"
+                }
+            }, 
+            "tool_id": "toolshed.g2.bx.psu.edu/repos/earlhaminst/blast_parser/blast_parser/0.1.2", 
             "tool_shed_repository": {
                 "changeset_revision": "363f3480622d", 
                 "name": "blast_parser", 
                 "owner": "earlhaminst", 
                 "tool_shed": "toolshed.g2.bx.psu.edu"
             }, 
-            "tool_state": "{\"input\": \"{\\\"__class__\\\": \\\"RuntimeValue\\\"}\", \"__rerun_remap_job_id__\": null, \"reciprocal\": \"\\\"true\\\"\", \"__page__\": null}", 
+            "tool_state": "{\"input\": \"{\\\"__class__\\\": \\\"ConnectedValue\\\"}\", \"__rerun_remap_job_id__\": null, \"reciprocal\": \"\\\"true\\\"\", \"__page__\": null}", 
             "tool_version": "0.1.2", 
             "type": "tool", 
             "uuid": "3eb98d54-a52d-4605-b068-28a700290d92", 
-            "workflow_outputs": [
-                {
-                    "label": null, 
-                    "output_name": "output", 
-                    "uuid": "6473f6a6-9036-43f2-ba10-961cecfb351f"
-                }
-            ]
+            "workflow_outputs": []
```

Comment:
I think we want to keep this as workflow output since it's used by the "Finding orphan genes" workflow.